### ISSUE TITLE
EnvelopeApi.listStatus(accountId, null, callback) Example

### ISSUE THREAD

**Issue by ECDonelson** [docusign/docusign-node-client#91]
Is there an example of the other api calls or is what's in the example it?

I'm looking to get the status of my documents from docusign and I've sent the request through (as I believe it should be) but I get a 400 error each time.

Has anyone looked into this, done this, is there any other examples out there that use this package?

**Comment by sangaloma** [comment]
It will be hard to tell without looking at your code, but I think you should go with listStatus(accountId, {}, callback)

**Comment by mmallis87** [comment]
@ECDonelson I agree with @sangaloma, please share your code as well as the printout of the error message that is returned to you by the callback function.
